10/03/2018 21:34:59.0118	[raft_python_callback.py write_to_logger 70]  118 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:init_state_functions_handler, LINE:28
MESSAGE:start

10/03/2018 21:34:59.0118	[raft_python_callback.py write_to_logger 70]  118 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:init_state_functions_handler, LINE:70
MESSAGE:end

10/03/2018 21:34:59.0118	[raft_python_callback.py write_to_logger 70]  118 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:init_multicast_message, LINE:81
MESSAGE:init multicast socket

10/03/2018 21:34:59.0119	[raft_python_callback.py write_to_logger 70]  119 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:init_multicast_message, LINE:88
MESSAGE:multicast was opened successfully

10/03/2018 21:34:59.0119	[raft_python_callback.py write_to_logger 70]  119 root [DEBUG] 
FILE:./C/Code/static_queue.c, FUNCTION:init_raft_queue, LINE:13
MESSAGE:first time

10/03/2018 21:34:59.0119	[raft_python_callback.py write_to_logger 70]  119 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=status 

10/03/2018 21:34:59.0119	[raft_python_callback.py write_to_logger 70]  119 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=-3 

10/03/2018 21:34:59.0120	[raft_python_callback.py write_to_logger 70]  120 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:init_raft, LINE:190
MESSAGE:init succeed

10/03/2018 21:34:59.0120	[raft_python_callback.py write_to_logger 70]  120 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_raft, LINE:231
MESSAGE:start raft...
VARIABLES:  raft_ip=230.1.1.10 

10/03/2018 21:34:59.0120	[raft_python_callback.py write_to_logger 70]  120 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_raft, LINE:233
MESSAGE:start raft...
VARIABLES:  raft_port=12345  server_id=2  members_num=2  leader_timeout=100 

10/03/2018 21:34:59.0120	[raft_python_callback.py write_to_logger 70]  120 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_raft, LINE:239
MESSAGE:run raft...

10/03/2018 21:34:59.0121	[raft_python_callback.py write_to_logger 70]  121 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:133
MESSAGE:try to read message

